pkg/validation/strfmt/default.go

```diff
@@ -205,6 +247,15 @@ func init() {
 
 	pw := Password("")
 	Default.Add("password", &pw, func(_ string) bool { return true })
+
+	dns1123subdomain := DNS1123Subdomain("")
+	Default.Add("dns1123subdomain", &dns1123subdomain, IsDNS1123Subdomain)
+
+	dns1123label := DNS1123Label("")
+	Default.Add("dns1123label", &dns1123label, IsDNS1123Label)
+
+	dns1035label := DNS1035Label("")
+	Default.Add("dns1035label", &dns1035label, IsDNS1035Label)
```

Comment:
if these get added to the default set of recognized formats here, how do we roll them out in a controlled way in kubernetes?

Reply:
This was held until CRD Ratcheting was promoted to beta in 1.30, we'll leverage that capability.

Reply:
I'll update this code so we can manage the newly added formats separate from the pre-existing ones.